docs/earth-age.rst

```diff
@@ -25,23 +25,23 @@ for pixel-registered grids; gridline-registered grids increment dimensions by on
 
 .. _tbl-earth_age:
 
-.. table:: EarthByte Global Earth Seafloor Crustal Age. An asterisk denotes tiled datasets.
+.. table:: EarthByte Global Earth Seafloor Crustal Age.
```

Comment:
Not directly related to this PR, but as "Global" was removed from the names of the remote datasets in PR #98, maybe it should be removed also here? 
```suggestion
.. table:: EarthByte Earth Seafloor Crustal Age.
```